src/components/Page/metaContent.tsx

```diff
@@ -63,8 +69,167 @@ export default function MetaContent({
           setMenuIsOpen={setMenuIsOpen}
           buttonsRef={buttonsRef}
         ></Menu>
+
         <ContentStyle menuIsOpen={menuIsOpen}>
           <div>
+            <LinkCardCollection>
+              {' '}
+              {/* Todo remove these added for testing purpose */}
+              <LinkCard isExternal={true} href={''} imageUrl={''} imgAltText="">
+                {'CardData.name'}
+              </LinkCard>
+              <LinkCard isExternal={true} href={''} imageUrl={''} imgAltText="">
+                {'CardData.name'}
+              </LinkCard>
+              <LinkCard isExternal={true} href={''} imageUrl={''} imgAltText="">
+                {'CardData.name'}
+              </LinkCard>
+              <LinkCard isExternal={true} href={''} imageUrl={''} imgAltText="">
+                {'CardData.name'}
+              </LinkCard>
+            </LinkCardCollection>
+            <PlatformFeatures platform="Javascript">
+              <CategoryFeature category="Auth">
+                <CategoryFeatureLink
+                  isExternal={true}
+                  href="https://docs.amplify.aws/"
+                >
+                  {'Enable Signup sign-in, and sign-out'}
+                </CategoryFeatureLink>
+                <CategoryFeatureText>
+                  {
+                    'Lorem Ipsum is simply dummy text of the printing and typesetting industry.'
+                  }
+                </CategoryFeatureText>
```

Comment:
What's the reason for separating out the `CategoryFeatureText` and `CategoryFeatureLink` into different components?

Reply:
I was matching the expected developer experience, I can change it to a single component.

Reply:
Sorry, what do you mean by the that Sowmya? I'm not sure what the expected developer experience is, so if it makes sense to split them out then that's okay with me

I was just thinking that since the `CategoryFeatureLink` is related to the `CategoryFeatureText` then it would make sense to have them grouped together

Let me know what you think though